beets/library.py

```diff
@@ -283,21 +287,24 @@ def key(x):
 # Exceptions.
 class FileOperationError(Exception):
     """Indicates an error when interacting with a file on disk.
+
     Possibilities include an unsupported media type, a permissions
     error, and an unhandled Mutagen exception.
     """
 
     def __init__(self, path, reason):
-        """Create an exception describing an operation on the file at
+        """Creates an exception describing an operation on the file at
```

Comment:
I notice that Google guide explicitly says to use descriptive verbs, which directly contradicts PEP 257's recommendation to use an imperative mood. Weird! That is going to be a _very_ hard habit for me to break, personally… and it even makes me question whether we really do want to embrace the Google style wholeheartedly.

Reply:
I would be fine with going back and changing the tense if that would be preferred! Just let me know if that is something I should do - I can go with whatever style is wanted.

Reply:
Sure! Why don't we go back to imperative verbs while we discuss a broader policy elsewhere?